src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -1236,7 +1257,7 @@ <h2 class="heading-medium">
           <strong>Role:</strong> {{ signerRole }}
           {% endif %}
 
-          <br/>{{ defenceSubmittedOn }}
+          <br/><span id="defenceSubmittedOn">{{ defenceSubmittedOn }}</span>
```

Comment:
An HTML document should only have unique `id`s. (Repeated on line [928](https://github.com/hmcts/cmc-claim-store/pull/1612/files#diff-1c9530b46501cb30d2f915bea3c5dcafR928).) Are either of them necessary?

Reply:
Removed until next PR